web/src/components/findings/FindingsDataTable.tsx

```
@@ -0,0 +1,427 @@
+
+  // Sort filtered findings based on current sort state
+  const sortedFindings = [...filteredFindings].sort((a, b) => {
+    if (!sortColumn || !sortDirection) return 0
+
+    let aValue: string | number
+    let bValue: string | number
+
+    switch (sortColumn) {
+      case 'severity':
+        // Sort by severity level (CRITICAL > HIGH > MEDIUM > LOW)
+        const severityOrder = { CRITICAL: 4, HIGH: 3, MEDIUM: 2, LOW: 1 }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)